### ISSUE TITLE
VLC 3.0 quits when doing  : Playback -> Renderer

### ISSUE THREAD

**Issue by sheepdestroyer** [flathub/org.videolan.VLC#23]
Hi,

I just installed VLC form flathub in fedora 27 x86-64
VLC quits silently each time I click on "Playback" then move the cursror over "Renderer".

Native fedora's VLC (from RPMfusion) does not quit like that (but Chromecast is sitll not working there either, showing Selected "Local" and greyed "Scanning..." in "Rendered" submenu)

Full log, from launching VLC to going to Renderer submenu : 
````
[sheepdestroyer@sheepora-X230 ~]$ /usr/bin/flatpak run -v --branch=stable --arch=x86_64 --command=/app/bin/vlc --file-forwarding org.videolan.VLC 
F: Opening user flatpak installation at path /home/sheepdestroyer/.local/share/flatpak
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/sheepdestroyer/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Allowing host-fs access
F: Allowing x11 access
F: Allowing pulseaudio access
VLC media player 3.0.0 Vetinari (revision 3.0.0-30-gef4c265336)
[0000000001b43830] main libvlc: Running vlc with the default interface. Use 'cvlc' to use vlc without interface.

(process:2): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.
Gtk-Message: Failed to load module "pk-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
Gtk-Message: Failed to load module "pk-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
[0000000001bd1280] main playlist: playlist is empty
[sheepdestroyer@sheepora-X230 ~]$ 
````

**Comment by MatMaul** [comment]
My guess is this bug on libmicrodns that I reported:
https://trac.videolan.org/vlc/ticket/19875
Should be fixed with next update #26.

**Comment by MatMaul** [comment]
Fixed.
